execution/executor/src/tests/mod.rs

```diff
@@ -145,15 +145,31 @@ fn test_executor_status() {
         .execute_block((block_id, block(vec![txn0, txn1, txn2])), parent_block_id)
         .unwrap();
 
-    assert_eq!(
-        &vec![
-            KEEP_STATUS.clone(),
-            KEEP_STATUS.clone(),
-            DISCARD_STATUS.clone(),
-            KEEP_STATUS.clone(),
-        ],
-        output.compute_status()
-    );
+    #[cfg(not(feature = "consensus-only-perf-test"))]
```

Comment:
nit: this feels kinda messy. Why not just ignore the entire test (like you've done above)?